tasks/restore_backup.yml

```diff
@@ -97,3 +97,8 @@
   with_items:
     - "OVESETUP_PKI/renew=bool:{{ he_pki_renew_on_restore }}"
     - "QUESTION/1/OVESETUP_SKIP_RENEW_PKI_CONFIRM=str:yes"
+- name: clear version lock from the engine
+  file:
+    state: touch
```

Comment:
shoildn't this be absent?

Reply:
absent would also work , but touch is empty the file   imo its better in this case

Reply:
done